File: Prime Number/Prime Number.cpp
#include<iostream>
using namespace std;

int main(){
    int num;
    cout<<"Please enter  number ==> ";
    cin>>num;
    int flag = 1;
    for(int i = 2;i<num;i++){
        if(num % i == 0){
            flag = 0;
        }
    }

    if(flag == 0){
        cout<<"Give number is not a prime number";
    }else{
        cout<<"Give number is prime number";
    }

    return 0;

}

File: Prime Number/Prime Number.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 18:28:48 2020

@author: ravi
"""

n = int(input("Please enter number ==> "))
flag = 1
for i in range(2,n):
    if n%i == 0:
        flag = 0
        break
if flag:
    print("Prime Number")
else:
    print("Not a Prime Number")